lib/utils/rruleUtil.js

```diff
@@ -0,0 +1,63 @@
+'use strict'
+
+let packageFound = false;
+try {
+  require('rrule');
+  packageFound = true;
+}
+catch (e) {
+  console.log('no package found');
+}
+
+let tryParseRRuleString = function(str) { return str;}
+let isRRuleObject = function(obj) {return false;}
+let nextInvocationDate = function(rrule, afterDate) {return null;}
+let doesRuleRecurr = function(rrule) {return false};
+
+
+if(packageFound) {
+  const {rrulestr, RRule, RRuleSet} = require('rrule');
+  /**
+   * Attempt to parse a string to a RRule object
+   * 
+   * @param {String} str the string to the parsed
+   * @returns A RRuleSet or RRule object if able to parse, otherwise will return the supplied string
+   */
+  tryParseRRuleString = function(str) {
+    try {
```

Comment:
This is not explicit enough. User should be very explicitly trying to add an RRule for us to try to parse it as one, and when that happens, we throw an error if it's not actualy an RRule.

Reply:
Yep, valid point ill change it